sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -214,12 +214,16 @@ System.out.printf("Updated Certificate with name %s and enabled status %s", upda
 
 ### Delete a Certificate
 
-Delete an existing Certificate by calling `deleteCertificate`.
+Delete an existing Certificate by calling `beginDeleteCertificate`.
 
 ```Java
-DeletedCertificate deletedCertificate = certificateClient.deleteCertificate("certificateName");
-System.out.printf("Deleted certificate with name %s and recovery id %s", deletedCertificate.getName(),
-    deletedCertificate.getRecoveryId());
+SyncPoller<DeletedCertificate, Void> deletedCertificatePoller =
```

Comment:
Nit: should this maybe be `deleteCertificatePoller`, since it's polling on the status of your call to `beginDeleteCertificate`?